.github/workflows/e2e-online-shop.yml

```diff
@@ -42,17 +58,34 @@ jobs:
           username: ${{ secrets.DOCKER_HUB_USERNAME }}
           password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
 
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
       - name: Set up Docker Buildx
         id: buildx
         uses: docker/setup-buildx-action@v2
 
+
       - name: Build and push
-        working-directory: benchmarks/online-shop/
-        env:
-          GOPRIVATE_KEY: ${{ secrets.EASE_LAB_ACCESS_KEY }}
-        run: |
-          make all
-          make push
+        if: matrix.service != 'cartservice'
```

Comment:
please add a comment explaining why it is omitted

Reply:
Good hit. Ill make it differently and add a context to the matrix